core/modules/simpletest/tests/database_test.module

```diff
@@ -206,6 +205,9 @@ function database_test_theme_tablesort($form, &$form_state) {
 
   $query = db_select('users', 'u');
   $query->condition('u.uid', 0, '<>');
+  // @todo: user_build_filter_query(), as well as user_filters() which is called
+  // by user_build_filter_query(), are slated for removal in Backdrop 2.0. We
+  // will need to build this query here manually when that happens.
   user_build_filter_query($query);
```

Comment:
This is not a change required from this issue.

Reply:
Fair enough 👍🏼 ...removed.